### ISSUE TITLE
New accessibility statement for new versions?

### ISSUE THREAD

**Issue by Ein-Tim** [corona-warn-app/cwa-website#3364]
## Your Question
* Website URL Path: https://www.coronawarn.app/en/accessibility/#ios & https://www.coronawarn.app/en/accessibility/#android
* Line / Paragraph: "This accessibility statement applies for version 2.17 of the Corona-Warn-App (iOS) of the Robert Koch Institute and Federal Ministry of Health." / "This accessibility statement applies for version 2.17 of the Corona-Warn-App (Android) of the Robert Koch Institute and Federal Ministry of Health.
* Question: The accessibility statement says that this statement applies to (IMO, "for" is wrong here) version 2.17 of the app. We are now at version 3.0. Will there be a new accessibility statement for this version? Or does the currently published statement apply to all versions starting from 2.17? If yes, this should be made clear in the texts, IMO.

---
Internal Tracking ID: [EXPOSUREAPP-14832](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14832)

**Comment by MikeMcC399** [comment]
@Ein-Tim 

Compare to German text from https://www.coronawarn.app/de/accessibility/#android

"Diese Erklärung zur Barrierefreiheit gilt für Version 2.17 der Corona-Warn-App (Android) des Robert Koch-Instituts und des Bundesministeriums für Gesundheit."

- PR https://github.com/corona-warn-app/cwa-website/pull/3360 already removed the text to say that there would be accessibility improvements.

---

You are right that  "This accessibility statement applies to version 2.17 ..." is grammatically better ("to" instead of "for")

**Comment by larswmh** [comment]
Thanks for raising this question @Ein-Tim. We have created an internal ticket for it and will raise this question internally.
Internal Tracking ID: [EXPOSUREAPP-14832](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14832)

---
Corona-Warn-App Open Source Team

**Comment by MikeMcC399** [comment]
@Ein-Tim 

Do you want to keep this open? If there is no more development and the app is taken out of the stores then there will no longer be any new accessibility statements.

**Comment by Ein-Tim** [comment]
@MikeMcC399 Yeah, you're right. Closing.
